src/core/account/account-api.ts

```diff
@@ -174,6 +175,27 @@ export function makeAccountApi(ai: ApiInput, accountId: string): EdgeAccount {
       return loginTree.username
     },
 
+    // ----------------------------------------------------------------
+    // Duress mode:
+    // ----------------------------------------------------------------
+
+    get canDuressLogin(): boolean {
+      const { activeAppId } = accountState()
+      const duressAppId = activeAppId.endsWith('.duress')
+        ? activeAppId
+        : activeAppId + '.duress'
```

Comment:
We do this calculation in many places. Could we pull it up to the top-level of `makeAccountApi`, so we only have to do it once and leave it in scope?